packages/docs/getting-started/running-a-validator.md

```diff
@@ -610,7 +610,17 @@ If everything goes well users should be able to see your claims by running:
 celocli account:get-metadata $CELO_VALIDATOR_ADDRESS
 ```
 
-By now, you should have setup your Validator account appropriately.
+You should see that your claim for `$CELO_VALIDATOR_GROUP_ADDRESS` could not be verified! We need to create the corresponding claim from `$CELO_VALIDATOR_GROUP_ADDRESS` otherwise anyone could claim it!
```

Comment:
We should note, very explicitly, that this is needed to have both your accounts appear on the leaderboard.

Furthermore, if people create additional accounts (e.g. when they add an additional validator), they will need to claim those accounts as well.

Please make sure this information is hard to miss